### ISSUE TITLE
[Screen reader - Make code - Settings]:  Incorrect accessible name is defined for the 'Settings' button.

### ISSUE THREAD

**Issue by vroopea** [microsoft/pxt-microbit#5490]
### User Experience:
Screen reader dependent people will get impacted if incorrect accessible name is defined for the settings button.  Due to which the end users will find difficulty in understanding the button and cannot access it quickly.

**Note:**
User credentials should NOT be included in the bug.

### Repro Steps:

1. Launch application URL : https://makecode.microbit.org/.
2. Home page will open. Turn on the screen reader (JAWS).
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to 'Settings' button and verify whether correct accessible name is defined or not.

### Actual Result:
**Issue:** Incorrect accessible name i.e., aria-label="Dropdown menu more..." is defined for the 'Settings' button.

**Observation:**
When focus lands on the 'Settings' button, the screen reader announces as 'Dropdown menu more...'.

### Expected Result:
Correct accessible name should be defined for the 'Settings' button.

**Example:** aria-label="Settings" should be defined.

### MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

### ARIA Authoring Ref Link/Fluent Patterns Guidance /HITS Guidance:
NA

### UIA/DOM details available:
Yes (Refer the attachment)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to an external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

### Test Environment:

- OS Version: Windows 11 23H2 (OS Build:22631.3007)
- Browser: Chrome Version 121.0.2277.4 (Official build) dev (64-bit).
- Test Matrix: Chrome + JAWS
- URL: https://makecode.microbit.org/.

<img width="960" alt="MAS2 4 6_Incorrect accessible name is defined for the 'Settings' button" src="https://github.com/microsoft/pxt-microbit/assets/158465724/cdad1c48-9564-4538-a003-b948872e9dd4">

**Comment by vroopea** [comment]
#Accessibility;#A11yMAS;#A11ySev2;#MAS2.4.6;#AILimited;#HCL;#Benchmark;#Win11-Chrome;#E2E-HCL-MakeCode-Feb2024;
